tests/test_bounce_lib.py

```diff
@@ -601,15 +601,15 @@ def test_crossover_bounce_no_existing_apps(self):
             "tasks_to_drain": set(),
         }
 
-    def test_crossover_bounce_old_but_no_new(self):
+    def test_crossover_bounce_old_app_but_no_new_app(self):
         """When marathon only has old apps for this service, crossover bounce should start the new one, but not kill any
         old tasks yet."""
 
-        new_config = {'id': 'foo.bar.12345', 'instances': 5}
+        new_config = {'id': 'foo.bar.12345', 'instances': 100}
```

Comment:
reverted the change.
I wanted to modify the test, but then I created a new one and forgot to revert my changes here.